webapp/src/blocks/workspace.ts

```diff
@@ -4,7 +4,7 @@ interface IWorkspace {
     readonly id: string,
     readonly title: string,
     readonly signupToken: string,
-    readonly settings: Readonly<Record<string, any>>
+    readonly settings: Readonly<Record<string, unknown>>
```

Comment:
This isn't right. In our backend, workspace.Settings is defined as `	Settings map[string]interface{} `json:"settings"`` so it can be anything, not unknown. You'll need to add a `// eslint-disable-next-line @typescript-eslint/no-explicit-any`